.github/workflows/build-mutating-webhook-helper.yaml

```diff
@@ -0,0 +1,50 @@
+name: Build mutating-webhook-helper
```

Comment:
mutating-webhook-helper is not used anymore.

We can exclude it from the github workflow.